pkg/controllers/deprovisioning/pdblimits.go

```diff
@@ -54,17 +54,17 @@ func NewPDBLimits(ctx context.Context, kubeClient client.Client) (*PDBLimits, er
 
 // CanEvictPods returns true if every pod in the list is evictable. They may not all be evictable simultaneously, but
 // for every PDB that controls the pods at least one pod can be evicted.
-func (s *PDBLimits) CanEvictPods(pods []*v1.Pod) bool {
+func (s *PDBLimits) CanEvictPods(pods []*v1.Pod) (client.ObjectKey, bool) {
```

Comment:
This seems like it could be modeled as an error, too.

Reply:
I know Karpenter does it elsewhere, but I don't like returning an error in non-error situations.